instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,10 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    enable_vm_dns_records = models.BooleanField(
+        default=False,
+        help_text="Set additional DNS records pointing directly to the active app servers."
+    )
```

Comment:
@smarnach True, re-reading the story I see how the fact that it was discussing the specific case of Harvard's instance could have made you consider it as a special case. I was describing a generic feature using their instance as an example, but this wasn't explicit.

Thanks for the PR! And I think we can leave the DNS records setup as a TODO in the code for now, no need for a story yet - if the behavior does become problematic we can revisit, but unless we end up hitting a cap on the domain names we can set in Gandi, it is harmless.